app/errors/error_service.ts

```diff
@@ -14,6 +15,10 @@ export class ErrorService {
   }
 
   handleError(e: any, options?: { ignoreErrorCodes: ErrorCode[] }) {
+    // NOTE: keep this string in sync with webtester.go -
+    // web tests check for this message and fail the test if it's logged.
+    console.error(new Error("Displaying error banner"));
+
     console.error(e);
```

Comment:
maybe i'm being unrealistic, but do we actually have any errors popping through at all? if not, could we actually just check the error level instead?

Reply:
We do have a lot of errors being logged unfortunately. I think that's a good idea though - it'd be nice to be able to do that instead, eventually.